app/helpers/additional_details/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module AdditionalDetails::ApplicationHelper
+
+  def convert_time(time)
+    modifier = ''
+    ((time.first == '0') || (time[0..1] == '10') || (time[0..1] == '11')) ? modifier = ' AM' : modifier = ' PM'
+
+    DateTime.strptime(time.gsub(':', ''), '%H%M').strftime('%l:%M') + modifier
```

Comment:
You could use `DateTime.strptime(time, '%H:%M').strftime('%l:%M %p')` to do the same thing all in 1 line